src/draw_target.rs

```diff
@@ -558,44 +587,34 @@ impl DrawState {
         let mut real_height = VisualLines::default();
 
         for (idx, line) in self.lines.iter().enumerate() {
-            let line_width = console::measure_text_width(line.inner());
-            let diff = if line.inner().is_empty() {
-                // Empty line are new line
-                1
-            } else {
-                // Calculate real length based on terminal width
-                // This take in account linewrap from terminal
-                let terminal_len = (line_width as f64 / term_width as f64).ceil() as usize;
-
-                // If the line is effectively empty (for example when it consists
-                // solely of ANSI color code sequences, count it the same as a
-                // new line. If the line is measured to be len = 0, we will
-                // subtract with overflow later.
-                usize::max(terminal_len, 1)
-            }
-            .into();
```

Comment:
Would prefer that changes to this code happen in a separate commit from moving it into a method.